doc/source/data/performance-tips.rst

```diff
@@ -7,14 +7,7 @@ Debugging Statistics
 ~~~~~~~~~~~~~~~~~~~~
 
 You can view debug stats for your Dataset and DatasetPipeline executions via ``ds.stats()``.
-These stats can be used to understand the performance of your Datasets workload and can help you debug problematic bottlenecks.
-
-At a high level, execution stats for tasks (e.g., CPU time) are attached to block metadata objects.
-Datasets have stats objects that hold references to these stats and parent dataset stats (this avoids stats holding references to parent datasets, allowing them to be garbage collected).
-Similarly, DatasetPipelines hold stats from recently computed datasets.
-In addition, we also collect statistics about iterator timings (time spent waiting / processing / in user code).
-Here's a sample output of getting stats in one of the most advanced use cases,
-namely iterating over a split of a dataset pipeline in a remote task:
+These stats can be used to understand the performance of your Dataset workload and can help you debug problematic bottlenecks. Note that both execution and iterator statistics are available:
```

Comment:
Can we briefly talk about how to read the stats?
For example, "Remote wall time": is it the time to complete all tasks in a stage, or just one of them? "Remote CPU time": similarly, is it the sum of all CPU costs across tasks or just a single one?

Reply:
Sure, I think that would be a good followup.